test/transform_test.py

```diff
@@ -1176,8 +1176,33 @@ def todo_test_smoothscale(self):
         #
         # New in pygame 1.8
 
-        self.fail()
-
+        #check stated exceptions
+        def smoothscale_low_bpp():
+            starting_surface = pygame.Surface((20, 20), depth=12)
+            smoothscaled_surface = pygame.transform.smoothscale(starting_surface, (10, 10))
+        self.assertRaises(ValueError, smoothscale_low_bpp)
+
+        def smoothscale_high_bpp():
+            starting_surface = pygame.Surface((20, 20), depth=48)
+            smoothscaled_surface = pygame.transform.smoothscale(starting_surface, (10, 10))
+        self.assertRaises(ValueError, smoothscale_high_bpp)
+
+        def smoothscale_invalid_scale():
+            starting_surface = pygame.Surface((20, 20), depth=36)
```

Comment:
should the depth here be 32 ?

Reply:
It definitely should be. This is not the value error we are looking for in this test! Thanks